### PR TITLE
Adjust examples for DALI v1.0 API

### PR DESCRIPTION
**Pull request by szalpal**
Adjusting examples for DALI's v1.0 recommended API (decorator)

### CODE REVIEW

File: docs/examples/identity/identity_pipeline.py [hunk 1]
```diff
@@ -20,20 +20,25 @@
...
 import nvidia.dali as dali
-import argparse
 
 
-def main(filename):
-    pipe = dali.pipeline.Pipeline(batch_size=3, num_threads=1, device_id=0)
-    with pipe:
-        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
-        pipe.set_outputs(data)
+def _parse_args():
```
Comment (mzient) [inline docs/examples/identity/identity_pipeline.py:25]: It's not a module, so I don't see a reason to make this function private.
```suggestion
def parse_args():
```

File: docs/examples/inception_ensemble/inception_pipeline.py [hunk 2]
```diff
@@ -21,28 +21,33 @@
...
-                                               dtype=types.FLOAT,
-                                               output_layout="HWC",
-                                               crop=(299, 299),
-                                               mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
-                                               std=[0.229 * 255, 0.224 * 255, 0.225 * 255])
-        pipe.set_outputs(images)
-        pipe.serialize(filename=filename)
+def _parse_args():
+    import argparse
+    parser = argparse.ArgumentParser(description="Serialize pipeline and save it to file")
```
Comment (mzient) [inline docs/examples/inception_ensemble/inception_pipeline.py:28]:
```suggestion
    parser = argparse.ArgumentParser(description="Serialize the pipeline and save it to a file")
```

File: docs/examples/inception_ensemble/inception_pipeline.py [hunk 3]
```diff
@@ -21,28 +21,33 @@
...
+    parser.add_argument('file_path', type=str, help='Path, where to save serialized pipeline')
```
Comment (mzient) [inline docs/examples/inception_ensemble/inception_pipeline.py:29]:
```suggestion
    parser.add_argument('file_path', type=str, help='The path where to save the serialized pipeline')
```

File: qa/L0_identity/model_repository/identity_pipeline.py [hunk 4]
```diff
@@ -20,20 +20,25 @@
...
 
 
-def main(filename):
-    pipe = dali.pipeline.Pipeline(batch_size=3, num_threads=1, device_id=0)
-    with pipe:
-        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
-        pipe.set_outputs(data)
+def _parse_args():
+    import argparse
+    parser = argparse.ArgumentParser(description="Serialize pipeline and save it to file")
```
Comment (mzient) [inline qa/L0_identity/model_repository/identity_pipeline.py:27]:
```suggestion
    parser = argparse.ArgumentParser(description="Serialize the pipeline and save it to a file")
```

File: qa/L0_identity/model_repository/identity_pipeline.py [hunk 5]
```diff
@@ -20,20 +20,25 @@
...
+    parser.add_argument('file_path', type=str, help='Path, where to save serialized pipeline')
```
Comment (mzient) [inline qa/L0_identity/model_repository/identity_pipeline.py:28]:
```suggestion
    parser.add_argument('file_path', type=str, help='The path, where to save the serialized pipeline')
```

File: qa/L0_inception_ensemble/model_repository/inception_pipeline.py [hunk 6]
```diff
@@ -21,28 +21,33 @@
...
-                                               dtype=types.FLOAT,
-                                               output_layout="HWC",
-                                               crop=(299, 299),
-                                               mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
-                                               std=[0.229 * 255, 0.224 * 255, 0.225 * 255])
-        pipe.set_outputs(images)
-        pipe.serialize(filename=filename)
+def _parse_args():
+    import argparse
+    parser = argparse.ArgumentParser(description="Serialize pipeline and save it to file")
```
Comment (mzient) [inline qa/L0_inception_ensemble/model_repository/inception_pipeline.py:28]:
```suggestion
    parser = argparse.ArgumentParser(description="Serialize the pipeline and save it to a file")
```

File: qa/L0_inception_ensemble/model_repository/inception_pipeline.py [hunk 7]
```diff
@@ -21,28 +21,33 @@
...
+    parser.add_argument('file_path', type=str, help='Path, where to save serialized pipeline')
```
Comment (mzient) [inline qa/L0_inception_ensemble/model_repository/inception_pipeline.py:29]:
```suggestion
    parser.add_argument('file_path', type=str, help='The path, where to save the serialized pipeline')
```

**Review by mzient — APPROVED** [review]

**Review by JanuszL — APPROVED** [review]

**Pull request merged**